engine/gfx/load_pics.asm

```diff
@@ -263,30 +263,30 @@ GLOBAL PICS_FIX
 	ret
 
 .PicsBanks: ; 511d4
-	db BANK(Pics_1) + 0
-	db BANK(Pics_1) + 1
-	db BANK(Pics_1) + 2
-	db BANK(Pics_1) + 3
-	db BANK(Pics_1) + 4
-	db BANK(Pics_1) + 5
-	db BANK(Pics_1) + 6
-	db BANK(Pics_1) + 7
-	db BANK(Pics_1) + 8
-	db BANK(Pics_1) + 9
-	db BANK(Pics_1) + 10
-	db BANK(Pics_1) + 11
-	db BANK(Pics_1) + 12
-	db BANK(Pics_1) + 13
-	db BANK(Pics_1) + 14
-	db BANK(Pics_1) + 15
-	db BANK(Pics_1) + 16
-	db BANK(Pics_1) + 17
-	db BANK(Pics_1) + 18
-	db BANK(Pics_1) + 19
-	db BANK(Pics_1) + 20
-	db BANK(Pics_1) + 21
-	db BANK(Pics_1) + 22
-	db BANK(Pics_1) + 23
+	db BANK("Pics 1") + 0
+	db BANK("Pics 1") + 1
+	db BANK("Pics 1") + 2
+	db BANK("Pics 1") + 3
+	db BANK("Pics 1") + 4
+	db BANK("Pics 1") + 5
+	db BANK("Pics 1") + 6
+	db BANK("Pics 1") + 7
+	db BANK("Pics 1") + 8
+	db BANK("Pics 1") + 9
+	db BANK("Pics 1") + 10
+	db BANK("Pics 1") + 11
+	db BANK("Pics 1") + 12
+	db BANK("Pics 1") + 13
+	db BANK("Pics 1") + 14
+	db BANK("Pics 1") + 15
+	db BANK("Pics 1") + 16
+	db BANK("Pics 1") + 17
+	db BANK("Pics 1") + 18
+	db BANK("Pics 1") + 19
+	db BANK("Pics 1") + 20
+	db BANK("Pics 1") + 21
+	db BANK("Pics 1") + 22
+	db BANK("Pics 1") + 23
```

Comment:
This should probably be `BANK("Pics 2")` and so on. The remaining sections could be empty. That might also make it so this function doesn't have to be immediately killed to work with pics.

Reply:
I did consider doing it that way from the start, but it seems like that would imply more flexibility than actually exists.

`FixPicBank` takes the "unfixed-bank" value from `dba_pic`, subtracts `BANK("Pics 1") - PICS_FIX`, and uses that as an index into `FixPicBank.PicsBanks`. If the array were `BANK("Pics 1")`, `BANK("Pics 2")`, etc, it would look like you can add `BANK("Pics N")` at the bottom and not worry about what N is; but in fact they'd still have to be contiguous since `N - (BANK("Pics 1") - PICS_FIX)` needs to be the corresponding array index.

(Also, the last five entries don't correspond to existing pic banks.)

Regardless, the whole thing is a design flaw no matter how it's written, so I've changed it. Just let me know if it should be different.